modules/nw-configure-ingress-access-logging.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+
+[id="nw-configure-ingress-access-logging_{context}"]
+= Configure Ingress access logging
+
+You can configure the Ingress Controller to enable access logs for the Ingress Controller. If you have clusters that do not receive much traffic, then you can log to a sidecar. If you have high traffic clusters, to avoid exceeding the capacity of the logging stack or integrate with a logging infrastructure outside of {product-title}, you can forward logs to a custom syslog endpoint. You can also specify the format for access logs.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+Configure Ingress access logging to a sidecar.
+
+* To configure Ingress access logging, you must specify a destination using `spec.logging.access.destination`. To specify logging to a sidecar destination, you must specify `container` `spec.logging.access.destination.type`. The following example is the definition of an Ingress Controller that logs to a `container` destination:
```

Comment:
it will be a little more intuitive to the user to mention that "when we create a sidecar container for logging, it creates a container named `logs`  inside the router namespace in question" and a supporting example like "it can be viewed with commands like" :
~~~
$ oc -n openshift-ingress logs  deployment.apps/router-internalapps -c logs
2020-04-24T04:27:51.158772+00:00 router-internalapps-6bd9c8f99c-m8bhs router-internalapps-6bd9c8f99c-m8bhs haproxy[30]: Proxy public started.
2020-04-24T04:27:51.158795+00:00 router-internalapps-6bd9c8f99c-m8bhs router-internalapps-6bd9c8f99c-m8bhs haproxy[30]: Proxy public_ssl started.
2020-04-24T04:27:51.158800+00:00 router-internalapps-6bd9c8f99c-m8bhs router-internalapps-6bd9c8f99c-m8bhs haproxy[30]: Proxy be_sni started.
2020-04-24T04:27:51.158814+00:00 router-internalapps-6bd9c8f99c-m8bhs router-internalapps-6bd9c8f99c-m8bhs haproxy[30]: Proxy fe_sni started.
~~~